src/components/GoalsCover.js

```diff
@@ -90,6 +95,9 @@ class GoalsCover extends React.Component {
       });
     }
 
+    const maxGoalsToShow =
+      window.screen.availWidth < 400 ? this.goals.length + 1 : 10;
```

Comment:
Could we use the window size? And recompute when the window size change?

Reply:
That’s the window size despite the name.

Recomputing when it changes would be nice. Let me look into this.

Reply:
Actually you are right. It’s  the width of the screen, not the window. Fixing this.